bin/docker-helper.sh

```diff
@@ -192,16 +200,11 @@ function cmd-push() {
       docker push $TARGET_IMAGE_NAME:$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION-$PLATFORM
     }
 
-    if [ -n "$FORCE_VERSION_TAG_PUSH" ] && [ "$FORCE_VERSION_TAG_PUSH" -eq "1" ]; then
```

Comment:
Removing this means we cannot force a tag push anymore, no?

Reply:
Yes, that's true, but this was only introduced to allow proper testing of the function in the bats tests. Afaik this isn't used anywhere.